lib/pages/portal/view/website_view.dart

```diff
@@ -90,9 +105,13 @@ class _WebsiteViewState extends State<WebsiteView> {
   }
 
   Website _buildWebsite() {
+    final String _id = _buildId();
```

Comment:
Leading underscores are only used to make class attributes private. This is a local variable, and there's no such thing as a "private" local variable, so please remove the leading underscore here and in the var below.

Reply:
On a second look, why make them variables at all? Please keep the previous formatting.

Reply:
I made a variable for it because i didn't want to call the method a second time for iconPath(id.ping).